hr_employee_lastnames/hook.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from odoo import SUPERUSER_ID
+from odoo.api import Environment
+
+
+def post_init_hook(cr, _):
+    cr.execute("UPDATE hr_employee SET firstname = NULL, lastname = NULL")
```

Comment:
there could be data set on firstname or lastname

Reply:
Yes, but is necessary to reprocess name a set firtname, lastname and lastname2 correctly